CHANGELOG.md

```diff
@@ -370,6 +371,7 @@
 * Exclude HID device: ELAN touch screen [#11324](https://github.com/mixxxdj/mixxx/pull/11324) [#11323](https://github.com/mixxxdj/mixxx/issues/11323)
 * Show otherwise hidden HID devices in developer mode  [#11317](https://github.com/mixxxdj/mixxx/pull/11317)
 * Use hidapi's hidraw backend instead of libusb on Linux [#4054](https://github.com/mixxxdj/mixxx/pull/4054)
+* Fix hid controllers: restore legacy passing of HID data in plain array [#11470](https://github.com/mixxxdj/mixxx/pull/11470) [#11461](https://github.com/mixxxdj/mixxx/issues/11461)
```

Comment:
What does this mean from a user's perspective?

Reply:
Mappings using hid-common-packet-parser.js aren't broken in 2.4. this fixes a regression introduced in refactoring that code.

Reply:
Ok, the new description is definitely better: https://github.com/mixxxdj/mixxx/pull/11568/commits/ad11a6463f4ccf3a386c54432cde6547507801ee